### ISSUE TITLE
Compatibility with altair 5.0.x

### ISSUE THREAD

**Issue by skupr-anaconda** [MaxHalford/prince#158]
`prince 0.10.7` requires `altair <5.0.0` but looking at the `altair 5.0.0` changelog  https://github.com/altair-viz/altair/blob/master/doc/releases/changes.rst#backward-incompatible-changes-1 it seems like the backward-incompatible changes do not impact the source code of the `prince` package. 
Is it correct? Can we bump the upper bound to **altair <6.0.0** or some changes are needed?

**Comment by MaxHalford** [comment]
Hey there. Sure, I think that's possible. I guess it's poetry which added `altair <5.0.0` by itself. Would you like to take of this in a pull request?

**Comment by skupr-anaconda** [comment]
First, I would test if it will work correctly. That's why I'm asking about the compatibility

**Comment by MaxHalford** [comment]
Of course. The way to test it would be to bump to dependency, and then run `jupyter nbconvert --execute --to notebook --inplace docs/content/*.ipynb` to check all the notebooks run without raising an exception. Then one would have to visually inspect the notebooks to checks nothing looks wrong.

**Comment by skupr-anaconda** [comment]
Looks like only ca.ipynb failed if I run:
`Visualization`
```
ca.plot(
    dataset,
    x_component=0,
    y_component=1
)
```
with an error:
```
Javascript Error: Failed to execute 'setAttribute' on 'Element': '0' is not a valid attribute name.
This usually means there's a typo in your chart specification. See the javascript console for the full traceback.
```
The same issue was on the `altair` project https://github.com/altair-viz/altair/issues/3041 but after reopening the notebook at the same browser, restarting the kernel with clearing cells, and running all cells again the issue has been resolved.

**Comment by skupr-anaconda** [comment]
I've opened a PR https://github.com/MaxHalford/prince/pull/159

**Comment by skupr-anaconda** [comment]
My logs:
`╰─❯ poetry install`
```
Installing dependencies from lock file

Package operations: 0 installs, 33 updates, 0 removals

  • Updating attrs (22.1.0 -> 23.1.0)
  • Updating platformdirs (2.5.2 -> 3.8.0)
  • Updating pyrsistent (0.18.0 -> 0.19.3)
  • Updating traitlets (5.7.1 -> 5.9.0)
  • Updating asttokens (2.0.5 -> 2.2.1)
  • Updating executing (0.8.3 -> 1.2.0)
  • Updating fastjsonschema (2.16.2 -> 2.17.1)
  • Updating pyzmq (25.0.2 -> 25.1.0)
  • Updating tornado (6.2 -> 6.3.2)
  • Updating wcwidth (0.2.5 -> 0.2.6)
  • Updating appnope (0.1.2 -> 0.1.3)
  • Updating jedi (0.18.1 -> 0.18.2)
  • Updating jupyter-client (8.1.0 -> 8.3.0)
  • Updating markupsafe (2.1.1 -> 2.1.3)
  • Updating nbformat (5.7.0 -> 5.9.0)
  • Updating numpy (1.24.2 -> 1.24.4)
  • Updating prompt-toolkit (3.0.36 -> 3.0.38)
  • Updating pygments (2.15.0 -> 2.15.1)
  • Updating pytz (2022.7 -> 2023.3)
  • Updating stack-data (0.2.0 -> 0.6.2)
  • Updating bleach (4.1.0 -> 6.0.0)
  • Updating comm (0.1.2 -> 0.1.3)
  • Updating debugpy (1.5.1 -> 1.6.7)
  • Updating jupyterlab-pygments (0.1.2 -> 0.2.2)
  • Updating mistune (0.8.4 -> 3.0.1)
  • Updating nbclient (0.5.13 -> 0.8.0)
  • Updating packaging (23.0 -> 23.1)
  • Updating pandas (1.5.3 -> 2.0.2)
  • Updating psutil (5.9.0 -> 5.9.5)
  • Updating virtualenv (20.17.1 -> 20.23.1)
  • Updating altair (4.2.2 -> 5.0.1)
  • Updating ipykernel (6.19.2 -> 6.23.3)
  • Updating nbconvert (6.5.4 -> 7.6.0)

Installing the current project: prince (0.10.7)

```

`╰─❯ poetry run jupyter nbconvert --execute --to notebook --inplace docs/content/*.ipynb`
```
[NbConvertApp] Converting notebook docs/content/ca.ipynb to notebook
[NbConvertApp] Writing 37696 bytes to docs/content/ca.ipynb
[NbConvertApp] Converting notebook docs/content/famd.ipynb to notebook
[NbConvertApp] Writing 138314 bytes to docs/content/famd.ipynb
[NbConvertApp] Converting notebook docs/content/gpa.ipynb to notebook
[NbConvertApp] Writing 18683 bytes to docs/content/gpa.ipynb
[NbConvertApp] Converting notebook docs/content/mca.ipynb to notebook
[NbConvertApp] Writing 35478 bytes to docs/content/mca.ipynb
[NbConvertApp] Converting notebook docs/content/mfa.ipynb to notebook
[NbConvertApp] Writing 29718 bytes to docs/content/mfa.ipynb
[NbConvertApp] Converting notebook docs/content/pca.ipynb to notebook
[NbConvertApp] Writing 104641 bytes to docs/content/pca.ipynb
```

**Comment by MaxHalford** [comment]
Ok great, it works for me too (locally). I've just published a new version (0.10.8) with your change. The docs are being built now.

Thanks!
